wizard/WizardCreateWalletFromDevice.qml

```diff
@@ -0,0 +1,125 @@
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import QtQuick 2.2
+import moneroComponents.WalletManager 1.0
+import moneroComponents.Wallet 1.0
+import QtQuick.Layouts 1.1
+import QtQuick.Dialogs 1.2
+import 'utils.js' as Utils
+
+ColumnLayout {
+    opacity: 0
+    visible: false
+
+    Behavior on opacity {
+        NumberAnimation { duration: 100; easing.type: Easing.InQuad }
+    }
+
+
+    onOpacityChanged: visible = opacity !== 0
+
+    function onWizardRestarted() {
+        // reset account name field
+        uiItem.accountNameText = defaultAccountName
+    }
+
+    //! function called each time we display this page
+
+    function onPageOpened(settingsOblect) {
+        checkNextButton()
+    }
+
+    function onPageClosed(settingsObject) {
+        settingsObject['account_name'] = uiItem.accountNameText
+        settingsObject['wallet_path'] = uiItem.walletPath
+        var restoreHeight = parseInt(uiItem.restoreHeight);
+        settingsObject['restore_height'] = isNaN(restoreHeight)? 0 : restoreHeight;
+        settingsObject['subaddress_lookahead'] = uiItem.subaddressLookahead;
+        var walletFullPath = wizard.createWalletPath(uiItem.walletPath,uiItem.accountNameText);
+        if (!wizard.walletPathValid(walletFullPath)) {
+            return false;
+        }
+        return createWalletFromDevice(settingsObject)
+    }
+
+    function checkNextButton() {
+        var wordsArray = Utils.lineBreaksToSpaces(uiItem.wordsTextItem.memoText).split(" ");
+        wizard.nextButton.enabled = wordsArray.length === 25;
+    }
+
+    //! function called each time we hide this page
+    //
+
+
+    function createWalletFromDevice(settingsObject) {
+        // TODO: create wallet in temporary filename and a) move it to the path specified by user after the final
+        // page submitted or b) delete it when program closed before reaching final page
+
+        // Always delete the wallet object before creating new - we could be stepping back from recovering wallet
+        if (typeof m_wallet !== 'undefined') {
+            walletManager.closeWallet()
+            console.log("deleting wallet")
+        }
+
+        var tmp_wallet_filename = oshelper.temporaryFilename();
+        console.log("Creating temporary wallet", tmp_wallet_filename)
+        var nettype = appWindow.persistentSettings.nettype;
+        var restoreHeight = settingsObject.restore_height;
+        var subaddressLookahead = settingsObject.subaddress_lookahead;
+
+        var wallet = walletManager.createWalletFromDevice(tmp_wallet_filename, "", nettype, "Ledger", restoreHeight, subaddressLookahead);
```

Comment:
Maybe make the "ledger" part a config/param, since the hw support code was made to be non ledger specific.